horace_core/sqw/coord_transform/@ortho_proj/ortho_proj.m

```diff
@@ -534,6 +534,38 @@
             flds = saveableFields@aProjectionBase(obj);
             flds = [flds(:);obj.fields_to_save_(:)];
         end
+        function [is,mess] = eq(obj,other_obj,varargin)
```

Comment:
Do not overload the `eq` (`==`) operator like this, the interface should be the same as the native/in-built function (`ok = eq(a,b)`), we already have `equal_to_tol` which defines this style of interface and should be used instead.

Reply:
well, we need this kind of overload in eq_to_toll. It is actively used there. The paren (in serializable) also have such interface.

And native overload also works, so I do not see reason not to use this kind of overloading unless you want two different eq functions with minor changes.

Reply:
Yes, I would much rather have two different equality functions rather than abusing MATLABs permissive overloading.